crypto/engine/eng_cryptodev.c

```diff
@@ -184,7 +186,6 @@ static struct {
     {
         CRYPTO_AES_CBC, NID_aes_256_cbc, 16, 32,
     },
-# ifdef CRYPTO_AES_CTR
```

Comment:
Note that there are multiple cryptodev implementations (openbsd/freebsd/linux cryptodev). The guard is probably written for the bsd one which has CRYPTO_AES_CTR as definition.